tests/upgrade_operator/upgrade_operator_suite_test.go

```diff
@@ -46,16 +46,13 @@ func TestLifecycle(t *testing.T) {
 func InstallOldOperator() {
 	step := "install old Cass Operator v1.1.0"
 	By(step)
-	err := helm_util.Install(oldOperatorChart, "cass-operator", ns.Namespace, map[string]string{})
-	mageutil.PanicOnError(err)
+	ginkgo_util.HelmInstallWithOverrides(oldOperatorChart, ns.Namespace, map[string]string{})
```

Comment:
So calling it in this way will result in the operator image potentially being overriden. This is a case where we want to _ignore_ the M_OPERATOR_IMAGE setting as the purpose of this test is to install an _old_ operator and then upgrade it.

Reply:
Ok, I will revert the changes to this test.